atom/errors.ts

```diff
@@ -7,9 +7,16 @@ store.listen(value => {
   value.value = 2
 })
 
+store.notify("value")
+
+// THROWS Argument of type '"nonExistentKey"' is not assignable to parameter of type '"value"'.
+store.notify("nonExistentKey")
+
 let fnStore = atom<() => void>(() => {
   fnStore.set(() => {})
 })
 
 let fn = fnStore.get()
 fn()
+
+fnStore.notify()
```

Comment:
Causes ts error for now? Should it be so?

Reply:
It means that all keys was changes. `changedKey` could be `undefined`.